### PR TITLE
Chtransfer multitransfer

### PR DESCRIPTION
**Pull request by andreevym**
methods for multi transfer operation, added new entity CCTransferItem this model contains fields token and amount

added new env MaxChannelTransferItems in ContractConfig by default 100

### CODE REVIEW

File: core/bc_chtransfer.go [hunk 1]
```diff
@@ -144,6 +194,93 @@ func (bc *BaseContract) createCCTransferFrom(
...
+	if len(items) == 0 || len(items) > bc.getMaxChannelTransferItems() {
+		return "", fmt.Errorf("%w found %d but expected from 1 to %d",
+			cctransfer.ErrInvalidTransferItemsCount, len(items), bc.getMaxChannelTransferItems(),
+		)
+	}
+
+	t := tokenSymbol(items[0].Token)
+	if !strings.EqualFold(bc.ContractConfig().GetSymbol(), t) && !strings.EqualFold(to, t) {
+		return "", cctransfer.ErrInvalidToken
+	}
```
Comment (pfi79) [inline core/bc_chtransfer.go:216]: maybe check all the tokens?

Comment (andreevym) [reply core/bc_chtransfer.go:216]: I check all tokens symbol is eq with symbol items[0]

File: test/unit/chtransfer_test.go [hunk 2]
```diff
@@ -2,6 +2,9 @@ package unit
 
 import (
 	"encoding/json"
+	"fmt"
+	"github.com/anoideaopen/foundation/core"
+	"github.com/anoideaopen/foundation/core/types/big"
 	"testing"
 	"time"
```
Comment (pfi79) [inline test/unit/chtransfer_test.go:9]: make go imports

File: test/unit/chtransfer_test.go [hunk 3]
```diff
@@ -80,6 +83,203 @@ func TestByCustomerForwardSuccess(t *testing.T) {
...
+	err = user1.InvokeWithError("vt", "channelTransferTo", id)
+	require.Error(t, err)
+
+	//user1.BalanceShouldBe("cc", 550)
+	user1.AllowedBalanceShouldBe("vt", "CC_1", 450)
+	user1.AllowedBalanceShouldBe("vt", "CC_2", 900)
+	//user1.CheckGivenBalanceShouldBe("vt", "VT", 0)
+	//user1.CheckGivenBalanceShouldBe("vt", "CC", 0)
+	//user1.CheckGivenBalanceShouldBe("cc", "CC", 0)
+	//user1.CheckGivenBalanceShouldBe("cc", "VT", 450)
```
Comment (pfi79) [inline test/unit/chtransfer_test.go:280]: unnecessary comments should be deleted or uncommented

File: test/unit/chtransfer_test.go (hunk 2 above)
Comment (andreevym) [reply test/unit/chtransfer_test.go]: fixed

File: test/unit/chtransfer_test.go (hunk 3 above)
Comment (andreevym) [reply test/unit/chtransfer_test.go]: fixed

**Comment by andreevym** [comment]
added for all multi transfer token test token because this is industrial token with group. renamed variable from cc and vt, to it1, it2

File: test/integration/channel-transfer-only-tx/channel_transfer_test.go [hunk 4]
```diff
@@ -4,10 +4,13 @@ import (
...
 	"path/filepath"
+	"strings"
 	"syscall"
 	"time"
 
-	"github.com/anoideaopen/foundation/proto"
+	"github.com/anoideaopen/foundation/core"
+	"github.com/anoideaopen/foundation/core/types/big"
+
 	pbfound "github.com/anoideaopen/foundation/proto"
```
Comment (pfi79) [inline test/integration/channel-transfer-only-tx/channel_transfer_test.go:14]: please exec go imports

Comment (andreevym) [reply test/integration/channel-transfer-only-tx/channel_transfer_test.go]: thanks, fixed

**Review by pfi79 — APPROVED** [review]

**Pull request merged**